docs/CONFIG.md

```diff
@@ -5035,6 +5075,16 @@ Depending on `SelectionMode`, this represents a difference in the number of bloc
 
 Set to 0 to disable this check.
 
+### LeaseDuration
+```toml
+LeaseDuration = '0s' # Default
+```
+LeaseDuration is the maximum period an internal service can use a node that is not considered the "best" node as
+defined by the SelectionMode. It also controls how often the lease check is done.
```

Comment:
Is this any easier to understand if we state it the other way around?
```suggestion
LeaseDuration is the minimum duration that the selected "best" node (as defined by SelectionMode) will be used, before re-selecting a better one if available. It also controls how often the lease check is done.
```
:shrug: 

And should we recommend a value? What is a 'low value'?

Reply:
+1 Jordan's rewording is easier for me to understand.